### DISCUSSION TITLE
Releases after 1.11.0 broke GLB loading that worked previously

### DISCUSSION THREAD

**Discussion by DurtyFree** [google/model-viewer#3605] [Q&A]
### Description

It seems like every release after 1.11.0 is not able to load a previously working GLB model.

Errors look as following: https://imgur.com/a/LvYONLo
```
THREE.GLTFLoader: Couldn't load texture blob:http://localhost:8081/1f74b052-d77c-4b46-a020-1df88543d5f8 [model-viewer.js:47542:11](https://unpkg.com/@google/model-viewer/dist/model-viewer.js)
TypeError: s is null
    assignTexture model-viewer.js:47595
[model-viewer.js:50001:24](https://unpkg.com/@google/model-viewer/dist/model-viewer.js)
viewer error: {"isTrusted":false} [localhost:8081:98:21](http://localhost:8081/)
THREE.GLTFLoader: Couldn't load texture blob:http://localhost:8081/bc6c28de-7197-4080-bdfb-9b5e3735c344 [model-viewer.js:47542:11](https://unpkg.com/@google/model-viewer/dist/model-viewer.js)
    h model-viewer.js:47542
THREE.GLTFLoader: Couldn't load texture blob:http://localhost:8081/415457a9-ee72-4d65-8dec-e3341d27865e [model-viewer.js:47542:11](https://unpkg.com/@google/model-viewer/dist/model-viewer.js)
    h model-viewer.js:47542
```

See attached zip for a GLB file, which can be successfully loaded in v1.11.0 but not in any version starting from v1.11.1.
[corsita.zip](https://github.com/google/model-viewer/files/9197672/corsita.zip)

### Version
- model-viewer: v1.11.1 - v.1.12.0

### Browser Affected
- [x] Chrome, 103.0.5060.134 (Official Build) (64-Bit) (cohort: Stable)
- [x] Edge
- [x] Firefox

### OS
- [x] Windows 10

**Comment by elalish** [comment]
I'm not sure how that model could have worked previously. The glTF [validator](https://github.khronos.org/glTF-Validator/) shows a bunch of errors about non-supported image formats.

**Reply by DurtyFree** [reply]
It worked before, so it seems like model-viewer just got more strict in these kinds of things. Anyway, I guess I will just have to stick to the older versions

**Reply by elalish** [reply]
I really don't think that's true; that file contains image data we do not have a parser for. I suppose it's possible it rendered and just lost those textures, but that's not exactly correct behavior either. What glTF exporter was used for those files?

**Reply by DurtyFree** [reply]
https://github.com/CesiumGS/obj2gltf this has been used

Yes its possible that the texture data might be lost / not rendering but atleast it shows the model with the rest, instead of not showing anything
